components/TextControls/BUILD

```diff
@@ -47,24 +48,38 @@ mdc_public_objc_library(
     ],
     deps = [
         "//components/AnimationTiming",
-        "@material_internationalization_ios//:MDFInternationalization",
         "//components/private/Math",
+        "@material_internationalization_ios//:MDFInternationalization",
+    ],
+)
+
+mdc_extension_objc_library(
+    name = "Theming",
+    deps = [
+        ":TextControls",
+        ":private",
```

Comment:
Why is `private` needed, shouldn't we only be theming public things? Also don't we need the subsystem schemes?

Reply:
Removed this dependency